entries/f/fidor.de.json

```
@@ -0,0 +1,19 @@
+{
+  "Fidor": {
```

```suggestion
  "Fidor Bank": {
```

They seem to use "Fidor Bank" on their website rather than just "Fidor".